### PR TITLE
[Modules] Update Synapse Worskpace to new dependencies approach

### LINKED ISSUE #2337: Update `Synapse/workspaces` to new dependencies approach
Ref to #1791 for guidelines

### PR DESCRIPTION
**Pull request by elbatane**
# Description

Updated Synapse workspace to the new dependency model. 

Changed 2 parameters to gain clarity on the module.

Fixed '\<tenantId\>'  within Service Fabric Cluster as I missed it when I changed the module to the dependency model.

Close #2337 

## Pipeline references

| Pipeline |
| - |
|[Synapse: Workspaces](https://github.com/Azure/ResourceModules/actions/workflows/ms.synapse.workspaces.yml) |
|[Service Fabric: Clusters](https://github.com/Azure/ResourceModules/actions/workflows/ms.servicefabric.clusters.yml)|

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

# Checklist

- [ ] I'm sure there are no other open Pull Requests for the same update/change
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (readme)
- [ ] I did format my code

### CODE REVIEW

File: modules/Microsoft.Synapse/workspaces/.bicep/nested_cmkRbac.bicep [hunk 1]
```diff
@@ -9,7 +9,7 @@ resource keyVault 'Microsoft.KeyVault/vaults@2019-09-01' existing = {
 
 // Assign RBAC role Key Vault Crypto User
 resource workspace_cmk_rbac 'Microsoft.Authorization/roleAssignments@2022-04-01' = if (usesRbacAuthorization) {
-  name: '${workspaceIdentity}-cmk-rbac'
+  name: guid('${workspaceIdentity}-cmk-rbac')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.bicep/nested_cmkRbac.bicep:12]: I'd probably suggest we align it with the other RBAC implementations like:

```bicep
name: guid('${keyVault.id}-${workspaceIdentity}-Key-Vault-Crypto-User')
```

File: modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep [hunk 2]
```diff
@@ -0,0 +1,100 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes')
+@maxLength(90)
+param resourceGroupName string = 'ms.synapse.workspaces-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep:10]:
```suggestion
@description('Optional. The location to deploy resources to.')
```

File: modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep:13]:
```suggestion
@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
```

File: modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep [hunk 4]
```diff
@@ -0,0 +1,56 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep:6]:
```suggestion
@description('Optional. The name of the resource group to deploy for testing purposes.')
```

File: modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+@maxLength(90)
+param resourceGroupName string = 'ms.synapse.workspaces-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep:10]:
```suggestion
@description('Optional. The location to deploy resources to.')
```

File: modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep:13]:
```suggestion
@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
```

File: modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep [hunk 7]
```diff
@@ -0,0 +1,57 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep:6]:
```suggestion
@description('Optional. The name of the resource group to deploy for testing purposes.')
```

File: modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+@maxLength(90)
+param resourceGroupName string = 'ms.synapse.workspaces-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep:10]:
```suggestion
@description('Optional. The location to deploy resources to.')
```

File: modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep:13]:
```suggestion
@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
```

File: modules/Microsoft.Synapse/workspaces/.bicep/nested_cmkRbac.bicep (hunk 1 above)
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.bicep/nested_cmkRbac.bicep:12]: Also, I know it's not part of the intended change, but could you rename `workspaceIdentity` to `workspaceIndentityPrincipalId`?

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep [hunk 10]
```diff
@@ -0,0 +1,45 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes')
+@maxLength(90)
+param resourceGroupName string = 'ms.synapse.workspaces-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:10]:
```suggestion
@description('Optional. The location to deploy resources to.')
```

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: '<<namePrefix>>${serviceShort}001'
+    defaultDataLakeStorageAccountName: 'adp<<namePrefix>>${serviceShort}001'
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:36]: I guess this dependency is missing?

File: modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep [hunk 12]
```diff
@@ -0,0 +1,40 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep:6]:
```suggestion
@description('Optional. The name of the resource group to deploy for testing purposes.')
```

File: modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+@maxLength(90)
+param resourceGroupName string = 'ms.synapse.workspaces-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to')
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep:10]:
```suggestion
@description('Optional. The location to deploy resources to.')
```

File: modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: '<<namePrefix>>${serviceShort}001'
+    defaultDataLakeStorageAccountName: 'adp<<namePrefix>>${serviceShort}001'
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep:47]: Missing dependency?

File: modules/Microsoft.ServiceFabric/clusters/.test/common/deploy.test.bicep [hunk 15]
```diff
@@ -58,7 +58,7 @@ module testDeployment '../../deploy.bicep' = {
     azureActiveDirectory: {
       clientApplication: resourceGroupResources.outputs.managedIdentityPrincipalId
       clusterApplication: 'cf33fea8-b30f-424f-ab73-c48d99e0b222'
```
Comment (AlexanderSehr) [inline modules/Microsoft.ServiceFabric/clusters/.test/common/deploy.test.bicep:60]: Must have missed this before? Do we know what this ID is `cf33fea8-b30f-424f-ab73-c48d99e0b222`?

If it is one of our own, I guess a dependency is missing. If it isn't we should at least mention what it is.

File: modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: '<<namePrefix>>${serviceShort}001'
+    defaultDataLakeStorageAccountName: 'adp<<namePrefix>>${serviceShort}001'
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep:59]: Missing dependency?

Comment (AlexanderSehr) [reply modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep:59]: I guess we could re-use the diagnosticsStorageAccount

File: modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep (hunk 14 above)
Comment (AlexanderSehr) [reply modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep:47]: I guess we could use the `diagnosticsStorageAccount`

File: modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: '<<namePrefix>>${serviceShort}001'
+    defaultDataLakeStorageAccountName: 'adp<<namePrefix>>${serviceShort}001'
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep:49]: Missing dependency?

Comment (AlexanderSehr) [reply modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep:49]: I guess we could use the `diagnosticsStorageAccount`

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep (hunk 11 above)
Comment (AlexanderSehr) [reply modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:36]: I guess we could use the `diagnosticsStorageAccount`

File: modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: '<<namePrefix>>${serviceShort}001'
+    defaultDataLakeStorageAccountName: 'adp<<namePrefix>>${serviceShort}001'
```
[the remark this answers is not in the document]
Comment (AlexanderSehr) [reply modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep:36]: I guess we could use the `diagnosticsStorageAccount`

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep (hunk 11 above)
Comment (eriqua) [reply modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:36]: This `defaultDataLakeStorageAccountName` makes me wonder if that needs to be a data lake storage.

File: modules/Microsoft.ServiceFabric/clusters/.test/common/deploy.test.bicep (hunk 15 above)
Comment (eriqua) [reply modules/Microsoft.ServiceFabric/clusters/.test/common/deploy.test.bicep:60]: From what I could grasp from [here](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-tutorial-create-vnet-and-windows-cluster#create-azure-ad-applications-and-assign-users-to-roles) it looks like a missing dependency on our side and I'm afraid not that quick to set up. Probably something to add to the manual dependencies still to be created?

Comment (AlexanderSehr) [reply modules/Microsoft.ServiceFabric/clusters/.test/common/deploy.test.bicep:60]: Agreed. As the AAD is involved there's not much wiggle room here

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep (hunk 11 above)
Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:36]: let me research a bit

Comment (AlexanderSehr) [reply modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:36]: You're right - fortunately its just another Storage Account (configured as a datalake)

Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:36]: Should I create then a storage acount with hns as dependecy?

File: modules/Microsoft.Synapse/workspaces/.bicep/nested_cmkRbac.bicep (hunk 1 above)
Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.bicep/nested_cmkRbac.bicep]: Done

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep (hunk 11 above)
Comment (AlexanderSehr) [reply modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:36]: Jap, that would be great :)

File: modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep (hunk 18 above)
[the remark this answers is not in the document]
Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep]: Done

File: modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep (hunk 14 above)
Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep]: Done

File: modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep (hunk 16 above)
Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep]: Done

File: .github/actions/templates/validateModulePester/action.yml [hunk 19]
```diff
@@ -141,8 +141,8 @@ runs:
           }
         }
 
-    - name: 'Publish Test Results'
-      uses: EnricoMi/publish-unit-test-result-action@v2
-      if: always()
-      with:
-        junit_files: '${{ steps.pester_run_step.outputs.outputPath }}'
+    # - name: 'Publish Test Results'
```
Comment (AlexanderSehr) [inline .github/actions/templates/validateModulePester/action.yml:144]: Should be undone for the main branch

File: modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep [hunk 20]
```diff
@@ -0,0 +1,106 @@
...
+param resourceGroupName string = 'ms.synapse.workspaces-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'swcom'
+
+@description('Optional. Data Lake Storage Filesystem.')
+param dataLakeStorageFilesystem string = 'synapsews'
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep:17]: In my opinion this should be hard-coded in the dependency deployment. Similar to the vnet subnet as they don't need to be unique

File: modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep [hunk 21]
```diff
@@ -0,0 +1,62 @@
...
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'swensa'
+
+@description('Generated. Used as a basis for unique resource names.')
+param baseTime string = utcNow('u')
+
+@description('Optional. Data Lake Storage Filesystem.')
+param dataLakeStorageFilesystem string = 'synapsews'
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep:20]: Same comment as in the other file

File: modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep [hunk 22]
```diff
@@ -0,0 +1,61 @@
...
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'swenua'
+
+@description('Generated. Used as a basis for unique resource names.')
+param baseTime string = utcNow('u')
+
+@description('Optional. Data Lake Storage Filesystem.')
+param dataLakeStorageFilesystem string = 'synapsews'
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep:20]: Also here, same comment

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep [hunk 23]
```diff
@@ -0,0 +1,56 @@
...
+param resourceGroupName string = 'ms.synapse.workspaces-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'swmanv'
+
+@description('Optional. Data Lake Storage Filesystem.')
+param dataLakeStorageFilesystem string = 'synapsews'
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:17]: And again, as in the other comment

File: modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep [hunk 24]
```diff
@@ -0,0 +1,51 @@
...
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.synapse.workspaces-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'swmin'
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep:14]: And again as in the other comment

File: modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep (hunk 20 above)
Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.test/common/deploy.test.bicep:17]: it is needed to create the container and as an input parameter that's why I added it as a parameter

File: modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep (hunk 22 above)
Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep]: changed

File: modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep (hunk 21 above)
Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.test/encrwsai/deploy.test.bicep]: changed

File: modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep (hunk 24 above)
Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep:14]: changed

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep (hunk 23 above)
Comment (elbatane) [reply modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep]: changed

File: .github/actions/templates/validateModulePester/action.yml (hunk 19 above)
Comment (elbatane) [reply .github/actions/templates/validateModulePester/action.yml]: Done

File: modules/Microsoft.Synapse/workspaces/.test/common/dependencies.bicep [hunk 25]
```diff
@@ -88,3 +87,6 @@ output privateDNSResourceId string = privateDNSZone.id
 
 @description('The name of the created Storage Account.')
 output storageAccountName string = storageAccount.name
+
+@description('The name of the created container.')
+output storageContainerName string = storageContainerName
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/common/dependencies.bicep:92]:
```suggestion
output storageContainerName string = storageAccount::blobService::container.name
```

Comment (AlexanderSehr) [reply modules/Microsoft.Synapse/workspaces/.test/common/dependencies.bicep:92]: Same for the other files

File: modules/Microsoft.Synapse/workspaces/.test/common/dependencies.bicep [hunk 26]
```diff
@@ -10,8 +10,7 @@ param virtualNetworkName string
 @description('Required. The name of the Storage Account to create.')
 param storageAccountName string
 
-@description('Required. The name of the Container to create.')
-param storageContainerName string
+var storageContainerName = 'synapsews'
```
[the remark this answers is not in the document]
Comment (AlexanderSehr) [reply modules/Microsoft.Synapse/workspaces/.test/common/dependencies.bicep:13]: Same for the other files

File: modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep [hunk 27]
```diff
@@ -31,7 +29,7 @@ module resourceGroupResources 'dependencies.bicep' = {
   name: '${uniqueString(deployment().name, location)}-paramNested'
   params: {
     storageAccountName: 'dep<<namePrefix>>sa${serviceShort}01'
-    storageContainerName: dataLakeStorageFilesystem
+
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/min/deploy.test.bicep:32]: Emtpy line

File: modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep [hunk 28]
```diff
@@ -31,7 +29,7 @@ module resourceGroupResources 'dependencies.bicep' = {
   name: '${uniqueString(deployment().name, location)}-paramNested'
   params: {
     storageAccountName: 'dep<<namePrefix>>sa${serviceShort}01'
-    storageContainerName: dataLakeStorageFilesystem
+
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/managedvnet/deploy.test.bicep:32]: Empty line

File: modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep [hunk 29]
```diff
@@ -37,7 +35,7 @@ module resourceGroupResources 'dependencies.bicep' = {
     keyVaultName: 'dep-<<namePrefix>>-kv-${serviceShort}-${substring(uniqueString(baseTime), 0, 3)}'
     managedIdentityName: 'dep-<<namePrefix>>-msi-${serviceShort}'
     storageAccountName: 'dep<<namePrefix>>sa${serviceShort}01'
-    storageContainerName: dataLakeStorageFilesystem
+
```
Comment (AlexanderSehr) [inline modules/Microsoft.Synapse/workspaces/.test/encrwuai/deploy.test.bicep:38]: Emtpy line

**Review by AlexanderSehr — APPROVED** [review]

**Pull request merged**